### ISSUE TITLE
Move to ica-sdk-python

### ISSUE THREAD

**Issue by alexiswl** [umccr/icav2-cli-plugins#214]
Replace libica with ica-sdk-python, wait on resolution of https://github.com/Illumina/ica-sdk-python/issues/1

**Comment by victorskl** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [ICA V2 Migration: Investigate ILMN ICA Python SDK comes with release v2.17](https://trello.com/c/VfzwrMhr/1504-ica-v2-migration-investigate-ilmn-ica-python-sdk-comes-with-release-v217)

**Comment by alexiswl** [comment]
Closing, superseded by #265
